contrib/assets_tutorial/bitcoin.conf

```diff
@@ -1,9 +1,14 @@
+regtest=1
+daemon=1
+txindex=1
+
+regtest.rpcuser=user3
+regtest.rpcpassword=password3
```

Comment:
I think these 2 lines are redundant because ```rpcuser``` and ```rpcpassword``` will apply to all networks if they are not in any section.

Reply:
I see - I will retest and amend accordingly. Thanks.

Reply:
LGTM.